Implementing Travis tests

jimbo8098 opened the pull request:
Adding basic travis tests to ensure the repo can build deps

jimbo8098 committed:
Implementing tests in Travis

jimbo8098 committed:
Removing specific branches used for testing

gsdevme commented on .travis.yml:
```diff
@@ -0,0 +1,5 @@
+language: ruby
+before_install:
+  - gem install bundler
```
worth using the makefile given we have one, so `make bundle`

gsdevme commented on Gemfile.lock:
```diff
@@ -0,0 +1,219 @@
+      bundler (>= 1.10)
+      chef-config (= 15.7.32)
+      chef-utils (= 15.7.32)
+      chef-zero (>= 14.0.11)
```
We maybe want to peg this to chef 13 though (at least right now).

gsdevme replied:
I guess there isn't any chef tests in here so doesn't matter too much

jimbo8098 committed:
- Using Chef 13.x
- Travis tests now using Makefile instead
- Readme added explaining the repo works with Cent7 and Chef 15.

jimbo8098 commented:
Both fixed in the last commit! Lemme know what you think.
